azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubClient.java

```diff
@@ -610,192 +610,38 @@ public final PartitionReceiver createReceiverSync(final String consumerGroupName
         }
     }
 
-    /**
```

Comment:
>/** [](start = 4, length = 3)

We need to clarify API removal. DotNET and CoreCLR are keeping existing APIs.

Reply:
We've talked about this, but leaving comments so everyone else can see. We're removing these APIs across all clients.